apps/03TicTacToe.js

```diff
@@ -24,23 +24,72 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  if (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn) {
+    console.log("horizontalWin");
+    return true;
+  } else if (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn) {
+    console.log("horizontalWin");
+    return true;
+  } else if (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn) {
+    console.log("horizontalWin");
+    return true;
+  }
 }
 
 function verticalWin() {
-  // Your code here
+  if (board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn) {
+    console.log("verticalWin");
+    return true;
+  } else if (board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn) {
+    console.log("verticalWin");
+    return true;
+  } else if (board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn) {
+    console.log("verticalWin");
+    return true;
+  }
 }
 
 function diagonalWin() {
-  // Your code here
+  if (board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn) {
```

Comment:
You're missing a diagonal win scenario here..

They're 8 total ways you can win at tic tac toe.